### PR TITLE
[ADD] sale_margin_percentage: New translated terms. vx#9981

### PR DESCRIPTION
**Pull request by deivislaya**
**TODO**

- [x] Modified translated terms

### CODE REVIEW

**Comment by deivislaya** [comment]
Hi @JulioSerna

This PR is ready for review

**Comment by deivislaya** [comment]
Hi @suniagajose 

This PR is ready for review

File: sale_margin_percentage/i18n/es.po [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+#: model:ir.model.fields,help:sale_margin_percentage.field_res_company_margin_threshold
+#: model:ir.model.fields,help:sale_margin_percentage.field_res_config_settings_margin_threshold
+msgid "Margin Threshold for product in order lines"
+msgstr "Límite de Margen para productos en lineas de venta"
+
+#. module: sale_margin_percentage
+#: model:ir.model.fields,help:sale_margin_percentage.field_sale_order_line_margin_percentage
+#: model:ir.model.fields,help:sale_margin_percentage.field_sale_order_margin_percentage
+msgid "Margin percentage compute based on price unit"
+msgstr "Cálculo del porcentaje de margen basado en la unidad de precio"
```
Comment (suniagajose) [inline sale_margin_percentage/i18n/es.po]: _... basado en precio unitario_

**Review by suniagajose — APPROVED** [review]
just a little change...

**Review by JulioSerna — APPROVED** [review]

**Pull request merged**